p2p/p2ptest/network.go

```diff
@@ -214,7 +222,7 @@ type Node struct {
 // MakeNode creates a new Node configured for the network with a
 // running peer manager, but does not add it to the existing
 // network. Callers are responsible for updating peering relationships.
-func (n *Network) MakeNode(t *testing.T) *Node {
+func (n *Network) MakeNode(t *testing.T, maxPeers, maxConnected uint16) *Node {
```

Comment:
Why not lean towards better flexibility in the future in this API by just accepting `NetworkOptions`?

Reply:
Sure, or perhaps there should be a `NodeOptions` that goes within `NetworkOptions`

Reply:
Yeah totally like the idea. Anything besides a long list of args that may change in the future 😆